lib/fluent/plugin/storage_local.rb

```diff
@@ -56,7 +56,7 @@ def configure(conf)
         elsif @path
           # ok
         else # @_plugin_id_configured is true
-          raise NotImplementedError, "implement this feature later with system_config"
+          raise NotImplementedError, "implement this feature later with system_config. Add <storage> section for file store or remove @id for memory store temporary"
```

Comment:
There are 2 ways for `in_dummy`.

- Add store_type like option to specify file or on-memory and log message in this line
- Fallback to on-memory storage, not raising `NotImplementedError`